src/lib/gas-refund-api.ts

```diff
@@ -106,7 +107,7 @@ export class GasRefundApi {
   async _fetchMerkleData(address: string): Promise<GasRefundClaim[]> {
     const grpData = await GasRefundParticipation.findAll({
       attributes: ['epoch', 'address', 'refundedAmountPSP', 'merkleProofs'],
-      where: { address, chainId: this.network, isCompleted: true },
+      where: { address, chainId: this.network },
```

Comment:
As we decided we won't store` refundedAmountPSP`  in GasRefundParticipation this will be undefined here. We gotta compute it via a join query involving GasRefundTransactions x GasRefundParticipations